### PR TITLE
Add methods to check column values

### PR DESCRIPTION
**Pull request by brianclio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,13 @@ example.state # 4
 
 example.state.save! # raises "ActiveRecord::RecordInvalid, 'Validation failed: State is not included in the list'"
 
-# You can add scopes on your enumerated column by passing in the scope name and an array containing values that should be included in your scope
+# You can check if an instance of your class has it's enumerated column set to a specific value.
```
Comment (fimmtiu) [inline README.md]: "its"

Comment (brianclio) [inline README.md]: derp

File: lib/rails_column_enumerator.rb [hunk 2]
```diff
@@ -38,6 +38,12 @@ def enumerated_column(column_name, enumerated_values, options = {})
       write_attribute(column_name, RailsColumnEnumerator.get_int(val, self.class.const_get(enum_name)))
     end
 
+    enumerated_values.each do |k, v|
```
Comment (fimmtiu) [inline lib/rails_column_enumerator.rb]: If you're not going to use `v`, I'd do `enumerated_values.keys.each` instead.

File: lib/rails_column_enumerator.rb [hunk 3]
```diff
@@ -38,6 +38,12 @@ def enumerated_column(column_name, enumerated_values, options = {})
...
+      self.send(:define_method, k.to_s + '?') do
+        k == self.send(column_name, true)
```
Comment (fimmtiu) [inline lib/rails_column_enumerator.rb]: How certain are you that the keys of the `enumerated_values` hash will be symbols? Could users pass in strings instead?

File: lib/rails_column_enumerator.rb [hunk 4]
```diff
@@ -38,6 +38,12 @@ def enumerated_column(column_name, enumerated_values, options = {})
       write_attribute(column_name, RailsColumnEnumerator.get_int(val, self.class.const_get(enum_name)))
     end
 
+    enumerated_values.keys.each do |key|
+      self.send(:define_method, key.to_s + '?') do
+        key.to_s == self.send(column_name, true).to_s
```
Comment (fimmtiu) [inline lib/rails_column_enumerator.rb]: Alternatively, could you not just do `key.to_sym == self.send(column_name, true)`?

Comment (brianclio) [inline lib/rails_column_enumerator.rb]: ... I could /facepalm

**Pull request merged**